tasties_app/tests/test_ingredient.py

```diff
@@ -0,0 +1,68 @@
+import pytest as pytest
+
+from tasties_app.models import Ingredient, Recipe, Category
+from django.contrib.auth.models import User
+
+
+@pytest.fixture
+def ingredient():
+    """
+    Creates a user, a recipes, and an ingredient,
+    and adds them to the database.
+    returns: ingredient
+    """
+    user1 = User.objects.create_user('testuser', 'password')
+    category = Category.objects.create(category_name='Breakfast')
+    category.save()
+    recipe = Recipe(title='Test Recipe1', author_id=user1, description='Test Description1',
+                    directions='Test Directions1', minutes_to_make=1, recipe_picture='test_picture1')
+    recipe.save()
+    recipe.categories.add(category)
+    ingredient = Ingredient(recipe_id=recipe, amount=1.0, measurement_unit='WHOLE',
+                            description='test ingredient')
+    ingredient.save()
+    return ingredient
+
+
+class TestIngredientModel:
+    """
+    Tests the Ingredient model:
+        - Creation of an ingredient
+        - Editing of an ingredient
+        - Negative tests to ensure that invalid data cannot be added to the database
+    """
+    @pytest.mark.django_db
+    def test_create_data_fixture(self, ingredient):
+        assert ingredient in Ingredient.objects.all()
+
+    @pytest.mark.django_db
+    def test_edit_ingredient(self, ingredient):
+        ingredient.amount = 2.0
+        ingredient.measurement_unit = "Fluid Ounce"
+        ingredient.description = "test2"
+        ingredient.full_clean()
+        ingredient.save()
+        assert ingredient.amount == 2.0
+        assert ingredient.measurement_unit == "Fluid Ounce"
+        assert ingredient.description == "test2"
+
+    @pytest.mark.django_db
+    def test_invalid_amount_input(self, ingredient):
+        with pytest.raises(Exception):
```

Comment:
We usually try to catch a specific exception whenever we can. This way you know that you got the error you wanted and didn't pass another exception